src/locales/cy/translation.json

```diff
@@ -267,25 +291,7 @@
         "subHeading2": "Os oes gennych ID gyda llun",
         "paragraph2": "Gallwch ddefnyddio ID gyda llun i brofi eich hunaniaeth gan ddefnyddio’r ap GOV.UK ID Check neu mewn Swyddfa Bost.",
         "details": {
-          "label": "Mathau o ID gyda llun y gallwch ei ddefnyddio",
-          "paragraph1": "For the GOV.UK ID Check app, you can use a:",
-          "appDocumentList": [
-            "UK photocard driving licence",
-            "UK passport",
-            "non-UK passport with a biometric chip",
-            "UK biometric residence permit (BRP)",
-            "UK biometric residence card (BRC)",
-            "UK Frontier Worker permit"
-          ],
-          "paragraph2": "At a Post Office, you can use a:",
-          "postOfficeDocumentList": [
-            "UK photocard driving licence",
-            "European Union (EU) photocard driving licence",
-            "UK passport",
-            "non-UK passport",
-            "National identity card from a European Economic Area (EEA) country",
-            "UK biometric residence permit (BRP)"
-          ]
+          "label": "Mathau o ID gyda llun y gallwch ei ddefnyddio"
```

Comment:
Minor preference of mine would be to unnest this and call it `detailsLabel`

Reply:
Good point